Problems bootstrapping repo

I want to contribute some changes to the Swift code generator, but having some trouble getting started as `npm install` seems to fail during the `npm run build` phase with the following error
`error TS2688: Cannot find type definition file for 'apollo-env'.`

<details>
  <summary>command output</summary>
<code><pre>
[...snip…]
> apollo-tooling-monorepo@ build /Users/johan/temp/apollo-tooling
> lerna run build --stream --scope apollo-env && npm run compile && lerna run build --stream --ignore apollo-env

lerna notice cli v3.13.1
lerna info versioning independent
lerna info filter [ 'apollo-env' ]
lerna info Executing command in 1 package: "npm run build"
apollo-env: > apollo-env@0.4.0 prebuild /Users/johan/temp/apollo-tooling/packages/apollo-env
apollo-env: > npm run clean
apollo-env: > apollo-env@0.4.0 clean /Users/johan/temp/apollo-tooling/packages/apollo-env
apollo-env: > npx rimraf lib
apollo-env: > apollo-env@0.4.0 build /Users/johan/temp/apollo-tooling/packages/apollo-env
apollo-env: > tsc --project ./src/fetch && node clone-types.js && cd src && tsc
apollo-env: copied: src/fetch/fetch.d.ts -> lib/fetch/fetch.d.ts
apollo-env: copied: src/fetch/global.d.ts -> lib/fetch/global.d.ts
apollo-env: copied: src/fetch/index.d.ts -> lib/fetch/index.d.ts
apollo-env: copied: src/fetch/url.d.ts -> lib/fetch/url.d.ts
lerna success run Ran npm script 'build' in 1 package in 3.1s:
lerna success - apollo-env

> apollo-tooling-monorepo@ compile /Users/johan/temp/apollo-tooling
> tsc --build tsconfig.json

error TS2688: Cannot find type definition file for 'apollo-env'.

error TS2688: Cannot find type definition file for 'apollo-env'.

error TS2688: Cannot find type definition file for 'apollo-env'.

error TS2688: Cannot find type definition file for 'apollo-env'.

error TS2688: Cannot find type definition file for 'apollo-env'.


Found 5 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! apollo-tooling-monorepo@ compile: `tsc --build tsconfig.json`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the apollo-tooling-monorepo@ compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/johan/.npm/_logs/2019-04-01T08_37_27_764Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! apollo-tooling-monorepo@ build: `lerna run build --stream --scope apollo-env && npm run compile && lerna run build --stream --ignore apollo-env`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the apollo-tooling-monorepo@ build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/johan/.npm/_logs/2019-04-01T08_37_27_800Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! apollo-tooling-monorepo@ postinstall: `npm run build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the apollo-tooling-monorepo@ postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/johan/.npm/_logs/2019-04-01T08_37_27_847Z-debug.log
</pre></code>

</details>

<details>
  <summary>2019-04-01T08_37_27_847Z-debug.log</summary>
<code><pre>
0 info it worked if it ends with ok
1 verbose cli [ '/usr/local/Cellar/node@10/10.15.2/bin/node',
1 verbose cli   '/usr/local/bin/npm',
1 verbose cli   'run',
1 verbose cli   'compile' ]
2 info using npm@6.9.0
3 info using node@v10.15.2
4 verbose run-script [ 'precompile', 'compile', 'postcompile' ]
5 info lifecycle apollo-tooling-monorepo@~precompile: apollo-tooling-monorepo@
6 info lifecycle apollo-tooling-monorepo@~compile: apollo-tooling-monorepo@
7 verbose lifecycle apollo-tooling-monorepo@~compile: unsafe-perm in lifecycle true
8 verbose lifecycle apollo-tooling-monorepo@~compile: PATH: /usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/Users/johan/temp/apollo-tooling/node_modules/.bin:/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/Users/johan/temp/apollo-tooling/node_modules/.bin:/Users/johan/Downloads/google-cloud-sdk/bin:/Users/johan/.rvm/gems/ruby-2.4.1/bin:/Users/johan/.rvm/gems/ruby-2.4.1@global/bin:/Users/johan/.rvm/rubies/ruby-2.4.1/bin:/Users/johan/.rvm/bin:/bin:/usr/local/bin:/sbin:/usr/local/lib:/usr/sbin:/usr/local/bin:/usr/local:/usr/bin:/bin:/Users/johan/bin:/Users/johan/.yarn/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin
9 verbose lifecycle apollo-tooling-monorepo@~compile: CWD: /Users/johan/temp/apollo-tooling
10 silly lifecycle apollo-tooling-monorepo@~compile: Args: [ '-c', 'tsc --build tsconfig.json' ]
11 silly lifecycle apollo-tooling-monorepo@~compile: Returned: code: 1  signal: null
12 info lifecycle apollo-tooling-monorepo@~compile: Failed to exec compile script
13 verbose stack Error: apollo-tooling-monorepo@ compile: `tsc --build tsconfig.json`
13 verbose stack Exit status 1
13 verbose stack     at EventEmitter.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:301:16)
13 verbose stack     at EventEmitter.emit (events.js:189:13)
13 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
13 verbose stack     at ChildProcess.emit (events.js:189:13)
13 verbose stack     at maybeClose (internal/child_process.js:970:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:259:5)
14 verbose pkgid apollo-tooling-monorepo@
15 verbose cwd /Users/johan/temp/apollo-tooling
16 verbose Darwin 18.5.0
17 verbose argv "/usr/local/Cellar/node@10/10.15.2/bin/node" "/usr/local/bin/npm" "run" "compile"
18 verbose node v10.15.2
19 verbose npm  v6.9.0
20 error code ELIFECYCLE
21 error errno 1
22 error apollo-tooling-monorepo@ compile: `tsc --build tsconfig.json`
22 error Exit status 1
23 error Failed at the apollo-tooling-monorepo@ compile script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 1, true ]
</pre></code>
</details>

--- a/package.json
+++ b/package.json
@@ -14,7 +14,7 @@
   },
   "scripts": {
     "azure": "npm ci && npm run test:ci",
-    "clean": "git clean -dfqX -- ./node_modules **/{lib,node_modules}/",
+    "clean": "git clean -dfqX -- ./node_modules **/{lib,node_modules}/ ./packages/*/tsconfig.tsbuildinfo",
     "postinstall": "npm run build",
     "build": "lerna run build --stream --scope apollo-env && npm run compile && lerna run build --stream --ignore apollo-env",
     "compile": "tsc --build tsconfig.json",

